### PR TITLE
Prep for publish

### PR DESCRIPTION
**Pull request by bdarfler**

### CODE REVIEW

**Comment by bdarfler** [comment]
@joshcough @jsolmon give a look see?

**Comment by jsolmon** [comment]
Looks good to me, assuming you've tested and the bintray uploads are successful. :+1:

**Comment by bdarfler** [comment]
Well not yet, I'm waiting on permissions from @joshcough. I'll wait on merging until I have them and have tested a publish to bintray.

File: build.sbt [hunk 1]
```diff
@@ -26,6 +28,18 @@ publishMavenStyle := false
...
+// https://github.com/softprops/bintray-sbt#publishing
+bintrayOrganization := Some("localytics")
+
+// Split staging from publishing
+// https://github.com/softprops/bintray-sbt#staging-optional
+bintrayReleaseOnPublish in ThisBuild := false
+
+// Bintray labels
+// https://github.com/softprops/bintray-sbt#labels
+bintrayPackageLabels := Seq("localytics", "sbt", "aws", "dynamodb", "test", "testing")
```
Comment (joshcough) [inline build.sbt]: :+1:

**Comment by joshcough** [comment]
:shipit:

**Pull request merged**